lguohan/sonic-buildimage
# [platform] delta-ag9032v1: update modules & debian package

Update delta_ag9032v1_platform.c to conform kernel > 4.7.

- **What I did**
Add new mux_select function for kernel > 4.7.
- **How I did it**
Adapt new i2c_mux APIs for kernel > 4.7.
- **How to verify it**
Build & check both in kernel 3.16 and 4.9.

Modify the ag9032v1 debian package for kernel > 4.7

- **What I did**
Add systemd for kernel 4.7 and later.
- **How I did it**
Add platform-modules-ag9032v1.service to call the platform service
- **How to verify it**
Build & check the functions in kernel 4.9.
